en/Live preview update.md

```diff
@@ -1,61 +1,60 @@
-Live Preview is introduced to help you preview what you're writing on the go, as well as hide Markdown formatting.
+Obsidian 13 introduces a brand new editor engine that comes with several enhancements to your editing experience:
 
-See Live Preview in action:
+- Live Preview
+- Spellchecker
+- Text substitution
+- Vim mode
 
-![[Live Preview.gif]]
+In this post, we'll take a closer look at one of the biggest improvements in the latest release: _Live Preview_.
 
-Source view of the same file:
+Live Preview is a new editor mode that lets you preview your notes in the same view that you're writing them in. Live Preview delivers a cleaner writing experience by only displaying Markdown syntax for the selected line. Let's see how it looks like!
```

Comment:
Technically in live preview only the markdown syntax around your cursor will be displayed, for example if you have 4 bold segments on one line, only the one you're editing will be displayed. Is there anyway to make it more clear here?